packages/passportjs-sample/src/app.ts

```diff
@@ -0,0 +1,66 @@
+import http from 'node:http';
+
+import cookieParser from 'cookie-parser';
+import express, { type Request } from 'express';
+import session from 'express-session';
+import passport from 'passport';
+
+import { config } from './config.js';
+import initPassport from './passport.js';
+
+const { appId, endpoint, cookieSecret } = config;
+
+initPassport();
+
+const app = express();
```

Comment:
So passport is based on express?

Reply:
Yes, "Passport can be unobtrusively dropped in to any [Express](https://expressjs.com/)-based web application"